<?xml version="1.0" encoding="UTF-8"?>
<launch>

        <arg name="R" default="30" />
        <arg name="G" default="30" />
        <arg name="B" default="30" />
        <arg name="colour_error" default="50.0" />
        <!-- mode if its different from colour_cal, it will start following line ( line_follow )-->
        <arg name="mode" default="follow_line" />

        <node name="line_follower_start"
            pkg="morpheus_chair_pkg"
            type="line_follower.py"
            respawn="false" output="screen" args="$(arg R) $(arg G) $(arg B) $(arg colour_error) $(arg mode)">
            <remap from="/morpheus_bot/raspicam_node/image_raw" to="raspicam_node/image/compressed" />
            <remap from="/morpheus_bot/cmd_vel" to="line_follower/cmd_vel" />

        </node>
</launch>